### PR TITLE
Fix documentation for the website

### PR DESCRIPTION
**Pull request by DamianArado**
Fixes: 
- https://github.com/microsoft/TypeScript-Website/issues/2075
- https://github.com/microsoft/TypeScript-Website/issues/2186
- https://github.com/microsoft/TypeScript-Website/issues/2229

### CODE REVIEW

**Review by SMAKSS — APPROVED** [review]

**Review by Harry-Hopkinson — APPROVED** [review]

**Comment by DamianArado** [comment]
Hi @orta, What do you think of these changes? 🤔

File: packages/documentation/copy/en/handbook-v2/More on Functions.md [hunk 1]
```diff
@@ -329,9 +329,9 @@ Functions in JavaScript often take a variable number of arguments.
 For example, the `toFixed` method of `number` takes an optional digit count:
 
 ```ts twoslash
-function f(n: number) {
+function f(n: number, x?: number) {
   console.log(n.toFixed()); // 0 arguments
-  console.log(n.toFixed(3)); // 1 argument
+  console.log(n.toFixed(x)); // 1 argument
```
Comment (jakebailey) [inline packages/documentation/copy/en/handbook-v2/More on Functions.md]: I don't think this is a good change; the optionality of the parameter to `toFixed` was already covered by either not passing an argument, or passing one, which is what this code was doing before.

File: packages/documentation/copy/en/handbook-v2/Type Manipulation/Generics.md [hunk 2]
```diff
@@ -357,8 +357,8 @@ class Animal {
   numLegs: number = 4;
 }
 
-class Bee extends Animal {
```
Comment (jakebailey) [inline packages/documentation/copy/en/handbook-v2/Type Manipulation/Generics.md]: Not to be a buzzkill (pun intended), but, I'm not super sure if this is the right change; reasonably the "fix" should have been to override `numLegs` with `numLegs = 6` in the `Bee` class, right? And that's why it was declared as `number` in `Animal`?

Also if we want to be animal pendants... cats shouldn't drink milk! 🐱

File: packages/documentation/copy/en/release-notes/TypeScript 1.1.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ oneline: TypeScript 1.1 Release Notes
 
 ## Performance Improvements
 
-The 1.1 compiler is typically around 4x faster than any previous release. See [this blog post for some impressive charts.](http://blogs.msdn.com/b/typescript/archive/2014/10/06/announcing-typescript-1-1-ctp.aspx)
+The 1.1 compiler is typically around 4x faster than any previous release. See [this blog post for some impressive charts.](https://www.infoq.com/news/2014/10/typescript-11-ctp-available/)
```
Comment (jakebailey) [inline packages/documentation/copy/en/release-notes/TypeScript 1.1.md]: Probably we should just link to http://web.archive.org/web/20141007020020/http://blogs.msdn.com/b/typescript/archive/2014/10/06/announcing-typescript-1-1-ctp.aspx at this point. That, or link to the actual blog post but fix the links.

**Review by jakebailey — CHANGES_REQUESTED** [review]

File: packages/documentation/copy/en/handbook-v2/Type Manipulation/Generics.md (hunk 2 above)
Comment (jakebailey) [reply packages/documentation/copy/en/handbook-v2/Type Manipulation/Generics.md]: #2846

**Review by jakebailey — APPROVED** [review]

**Comment by jakebailey** [comment]
Updated the PR to get it pushed forward.

**Pull request merged**